webapp/channels/src/components/widgets/tag/tag.tsx

```diff
@@ -108,6 +108,11 @@ const TagWrapper = styled.div<TagWrapperProps>`
         color: rgb(var(--semantic-color-danger));
     }
 
+    &.Tag--dim {
+        background: rgba(var(--semantic-color-general), 0.08);
+        color: rgb(var(--semantic-color-general));
+    }
+
```

Comment:
```suggestion
```
These shouldn't be needed since they are the default styles for the Tag component.

Reply:
see line 83 & 84

Reply:
see lin 83 and 84